CHANGELOG.md

```diff
@@ -7,10 +7,19 @@
 Special thanks to external contributors on this release:
 @danil-lashin, @guagualvcha, @jleni, @siburu, @silasdavis, @srmo, @Stumble, @svenstaro 
 
+This release brings pubsub 2.0, limits the mempool size to 1GB (max_txs_bytes) and number of `/subscribe` WebSocket 
+clients (`max_subscription_clients`) and adds `/unsubscribe_all` endpoint to the lite client. 
+It also contains many smaller improvements and bug-fixes. 
+Pubsub 2.0 is an improved version of the older pubsub, which is a) non-blocking b) has nicer API. 
+Note our HttpClient's interface got updated to reflect the pubsub changes and now also has a better API for WebSocket subscriptions.
+
 ### BREAKING CHANGES:
 
 * CLI/RPC/Config
 - [rpc/client] Update Subscribe interface to reflect new pubsub/eventBus API [ADR-33](https://github.com/tendermint/tendermint/blob/develop/docs/architecture/adr-033-pubsub.md)
+- [config] Add `rpc.max_subscription_clients` config parameter to control how many unique clientIDs can `/subscribe` at the same time
```

Comment:
\#2826

Reply:
Added link to this and the following 2 lines in: https://github.com/tendermint/tendermint/pull/3427/commits/284d9c874f2329f6b6fbd83b0009d8ca6e093c91